src/modules/0.0.14/aws_vpc/index.ts

```diff
@@ -495,19 +501,41 @@ export const AwsVpcModule: Module2 = new Module2({
     vpc: new Mapper2<Vpc>({
       entity: Vpc,
       equals: (a: Vpc, b: Vpc) => {
-        const result = Object.is(a.cidrBlock, b.cidrBlock) && Object.is(a.state, b.state) && Object.is(a.isDefault, b.isDefault);
+        const result = Object.is(a.cidrBlock, b.cidrBlock) && Object.is(a.state, b.state) && Object.is(a.isDefault, b.isDefault) &&
+          (AwsVpcModule.utils.eqTags(a.tags, b.tags));
         return result;
       },
       source: 'db',
       cloud: new Crud2({
+        updateOrReplace: (a: Vpc, b: Vpc) => {
+          if (Object.is(a.cidrBlock, b.cidrBlock)) return "replace";
```

Comment:
Hmm I think here it is the opposite, right? 
```suggestion
          if (!Object.is(a.cidrBlock, b.cidrBlock)) return "replace";
```

Reply:
Please also include the `isDefault` property for the `replace` path